lib/CL/devices/ventus/pocl_ventus.cc

```diff
@@ -1312,6 +1311,20 @@ pocl_ventus_copy_rect (void *data, pocl_mem_identifier *dst_mem_id,
     free(dst_buff);
 }
 
+void
+pocl_ventus_memfill (void *data, pocl_mem_identifier *dst_mem_id,
+                     cl_mem dst_buf, size_t size, size_t offset,
+                     const void *__restrict__ pattern, size_t pattern_size)
+{
+  struct vt_device_data_t *d = (struct vt_device_data_t *)data;
+  void *host_ptr = pocl_aligned_malloc(MAX_EXTENDED_ALIGNMENT, size);
```

Comment:
Why can't it be something like "char *buff = (char *)malloc(size);"? What is the principle of alignment here?